octodns/record/__init__.py

```diff
@@ -217,6 +217,27 @@ def healthcheck_port(self):
         except KeyError:
             return 443
 
+    @property
+    def healthcheck_statuscode(self):
+        healthcheck = self._octodns.get('healthcheck', {})
+        if healthcheck.get('protocol', None) == 'TCP':
+            return None
+        try:
+            return healthcheck['status_code']
+        except KeyError:
+            return 200
+
+    # string to check in response body
+    @property
+    def healthcheck_responsebody(self):
+        healthcheck = self._octodns.get('healthcheck', {})
+        if healthcheck.get('protocol', None) == 'TCP':
+            return None
+        try:
+            return healthcheck['response_body']
+        except KeyError:
+            return None
```

Comment:
Body implies that it's matching the full response body, but this is just searching for a string in the response (body) and thus `contains` is more appropriate. 


```suggestion
    # string to check for in response body
    @property
    def healthcheck_response_contains(self):
        healthcheck = self._octodns.get('healthcheck', {})
        if healthcheck.get('protocol', None) == 'TCP':
            return None
        try:
            return healthcheck['response_contains']
        except KeyError:
            return None
```